src/CommandHandler.cpp

```diff
@@ -414,7 +421,15 @@ void CommandHandler::handle(std::string const &Command) {
   if (auto x = find<std::string>("cmd", Doc)) {
     std::string CommandMain = x.inner();
     if (CommandMain == "FileWriter_new") {
-      handleNew(Command);
+      try {
+        handleNew(Command);
+      } catch (JobIdException &e) {
+        LOG(Sev::Critical, "{}", e.what());
+        throw JobIdException(e.what());
```

Comment:
Why throw the same error message via a fresh constructed exception? Why not just `throw` if the same exception should bubble further up?

Reply:
As we discussed e can let  ``tryToHandle()`` (instead of ``handle(std::string const &)``) handle any exception that ``handleNew`` throws.